example/Python/src/DigitalTwin/KeyFob/dev/README.md

```diff
@@ -28,30 +28,37 @@ NAME    ZONE           MACHINE_TYPE   PREEMPTIBLE  INTERNAL_IP  EXTERNAL_IP
 rti-vm  us-central1-a  n1-standard-1               10.128.0.7   34.133.143.163  RUNNING
 ```
 
+Export the `EXTERNAL_IP` of the RTI:
+```bash
+RTI_IP=`gcloud compute instances list | grep 'rti-vm' | awk '{print $5}'`
```

Comment:
Is there an `export` missing here?

Reply:
assignment `=` sets a variable local to the current bash process, so `export` is optional